docs/WritingTests.asciidoc

```diff
@@ -1408,3 +1408,9 @@ state of the specified git refspec. For example the asset caching
 functionality will override the +PRODUCTDIR+ variable. The above method can
 still be used in this case in the common case that the test schedule does not
 need to be changed or in case a custom schedule is defined by +SCHEDULE+.
+
+== Controling list of log files to upload
```

Comment:
s/Controling/Controlling/ and *the* list

Reply:
done